src/postgraphql/cli.ts

```diff
@@ -113,12 +123,56 @@ const {
   readCache,
   writeCache,
 // tslint:disable-next-line no-any
-} = program as any
+} = config['options'] as any
```

Comment:
This will throw if `config.options` isn't set (e.g. if you source an RC that is just `{}`); however, see recommendation below to fix this and remove a load of code in one fell swoop.